src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -240,103 +239,6 @@ public function hasAuthenticatedBackendUser()
 
 
 	/**
-	 * Add the template output to the cache and add the cache headers
-	 */
-	protected function addToCache()
```

Comment:
I use it [here](https://github.com/1up-lab/contao-cache-warmup/blob/master/src/CacheWarmup/CacheWarmup.php#L219) to generate cache files for desktop and mobile [independent from a mobile layout](https://github.com/contao/core/blob/master/system/modules/core/classes/FrontendTemplate.php#L233-L244).